packages/flutter_tools/test/general.shard/base/os_test.dart

```diff
@@ -496,6 +497,33 @@ void main() {
       );
       expect(utils.name, 'Pretty Name');
     });
+
+    // See https://snyk.io/research/zip-slip-vulnerability for more context
+    testWithoutContext('Windows validates paths when unzipping', () {
+      // on POSIX systems we use the `unzip` binary, which will fail to extract
+      // files with paths outside the target directory
+      final OperatingSystemUtils utils = createOSUtils(FakePlatform(operatingSystem: 'windows'));
+      final MemoryFileSystem fs = MemoryFileSystem.test();
+      final File fakeZipFile = fs.file('archive.zip');
+      final Directory targetDirectory = fs.directory('output')..createSync(recursive: true);
+      const String content = 'hello, world!';
+      final Archive archive = Archive()..addFile(
+        // This file would be extracted outside of the target extraction dir
+        ArchiveFile(r'..\..\..\target-file.txt', content.length, content.codeUnits),
```

Comment:
Can the file name in a zip file have a space in it? If so how about adding one?

Reply:
done